### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by leonglo** [Itroublve/Token-Browser-Password-Stealer-Creator#23]
I have put everything in the ItroublveTSC (webhook URL, bat and exe links, etc.) and created the token stealer.exe but once I launch it doesn't do anything. Please help

**Comment by Itroublve** [comment]
Try using the newer version in the newer repo, ItroublveTSC.

**Comment by leonglo** [comment]
I am already using the newest version but it doesn't work

**Comment by Itroublve** [comment]
> I am already using the newest version (V5.1)but it doesn't work 

Not in this repo, it's a different repository.

**Comment by Unawarding** [comment]
> > I am already using the newest version (V5.1)but it doesn't work
> 
> Not in this repo, it's a different repository.

Yeah but that one doesn't even have a .exe file. How do you start that?
Can't you put instructions in the "read me.md" file?
Would avoid many questions.
